src/sentry/conf/server.py

```diff
@@ -684,6 +684,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("replays.delete_replay", routing_key="replays.delete_replay"),
     Queue("counters-0", routing_key="counters-0"),
     Queue("triggers-0", routing_key="triggers-0"),
+    Queue("find-missing-codemappings", routing_key="find-missing-codemappings"),
```

Comment:
It seems we need to register it:
```
CeleryQueueRegisteredTest.test

AssertionError: Found tasks with queues that are undefined. These must be defined in settings.CELERY_QUEUES.
  Task Info:
   - Task: sentry.tasks.find_missing_codemappings, Queue: find_missing_codemappings.
assert not [' - Task: sentry.tasks.find_missing_codemappings, Queue: find_missing_codemappings']
```

Reply:
Actually I think it has to do with `find-missing-codemappings` vs `find_missing_codemappings`.